Program1.c

```diff
@@ -0,0 +1,144 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+struct User{
+    int id;
+    char name[100];
+    int age;
+};
+void create_user(){
+    FILE *file=fopen("users.txt","a");
+    if(file==NULL){
+        printf("Error opening file!\n");
+        return;
+    }
+    struct User user;
+    printf("Enter User ID: ");
+    scanf("%d",&user.id);
+    getchar();
+    printf("Enter Name: ");
+    scanf("%s",user.name);
+    printf("Enter Age: ");
+    scanf("%d",&user.age);
+    fprintf(file,"%d %s %d\n",user.id,user.name,user.age);
+    printf("User added successfully!\n");
+    fclose(file);
+}
+void read_users(){
+    FILE *file=fopen("users.txt","r");
+    if(file==NULL){
+        printf("Error opening file!\n");
+        return;
+    }
+    struct User user;
+    while(fscanf(file,"%d %s %d",&user.id,user.name,&user.age)!=EOF){
+        printf("ID: %d,Name: %s,Age: %d\n",user.id,user.name,user.age);
+    }
+    fclose(file);
+}
+void update_user(){

```

Comment:
The function doesn’t handle scenarios where the user inputs invalid data, such as letters for the age or ID.
Valid for other functions also.

Reply:
In updated user management branch code has been updated for age and ID.